### ISSUE TITLE
Failed to install add on 

### ISSUE THREAD

**Issue by doudigit** [davide125/hassio-addons#3]
for some reason installation of the add on fails and gives out this code 

`The command '/bin/sh -c sed -e 's:/homebridge/:/data/:g' -e 's: /homebridge: /data:g' -i /usr/bin/homebridge-* /usr/bin/with-localenv /etc/cont-init.d/45-user-data /etc/cont-init.d/50-plugins /etc/cont-init.d/55-config-ui /etc/logrotate.d/homebridge /etc/services.d/hb*/* /etc/services.d/homebridge*/*' returned a non-zero code: 2`

Log gives out this

```
Logger: homeassistant.components.hassio
Source: components/hassio/websocket_api.py:120 
Integration: Home Assistant Supervisor ([documentation](https://www.home-assistant.io/integrations/hassio), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+hassio%22)) 
First occurred: 19:45:28 (4 occurrences) 
Last logged: 19:49:32

Failed to to call /addons/b0cbb969_homebridge/install - The command '/bin/sh -c sed -e 's:/homebridge/:/data/:g' -e 's: /homebridge: /data:g' -i /usr/bin/homebridge-* /usr/bin/with-localenv /etc/cont-init.d/45-user-data /etc/cont-init.d/50-plugins /etc/cont-init.d/55-config-ui /etc/logrotate.d/homebridge /etc/services.d/hb*/* /etc/services.d/homebridge*/*' returned a non-zero code: 2
```

**Comment by WoodsyLawyer** [comment]
Same issue here when trying to install.  This is the version I'm running if it helps:

Home Assistant Core 2022.6.7
Home Assistant Supervisor 2022.05.3
Home Assistant OS 8.2

**Comment by Schwa-l-be** [comment]
`failed to get destination image "sha256:eb60f43b988dba040105c141f224776123e572103e08eab8a268af25c737fba4": image with reference sha256:eb60f43b988dba040105c141f224776123e572103e08eab8a268af25c737fba4 was found but does not match the specified platform: wanted linux/arm/v6, actual: linux/arm/v7`

I get the following error - I use a Pi 4

**Comment by martindinicolino** [comment]
> for some reason installation of the add on fails and gives out this code
> 
> `The command '/bin/sh -c sed -e 's:/homebridge/:/data/:g' -e 's: /homebridge: /data:g' -i /usr/bin/homebridge-* /usr/bin/with-localenv /etc/cont-init.d/45-user-data /etc/cont-init.d/50-plugins /etc/cont-init.d/55-config-ui /etc/logrotate.d/homebridge /etc/services.d/hb*/* /etc/services.d/homebridge*/*' returned a non-zero code: 2`


same here, also on a Pi4

**Comment by Mikaaaeru** [comment]
Hi did you guys fix you're issue ? 
i got the same on a pi 4 

### The command '/bin/sh -c sed -e 's:/homebridge/:/data/:g' -e 's: /homebridge: /data:g' -i /usr/bin/homebridge-* /usr/bin/with-localenv /etc/cont-init.d/45-user-data /etc/cont-init.d/50-plugins /etc/cont-init.d/55-config-ui /etc/logrotate.d/homebridge /etc/services.d/hb*/* /etc/services.d/homebridge*/*' returned a non-zero code: 2

**Comment by davide125** [comment]
This should have been fixed with #4.
